### PR TITLE
PAGER: split on " "

### PR DESCRIPTION
**Pull request by Profpatsch**
$PAGER could be something like `"less -R"`, and `-R` should be given as
argument, not as executable.

### CODE REVIEW

**Comment by nfnty** [comment]
Use `shlex.split()` instead.

File: ranger/core/actions.py [hunk 1]
```diff
@@ -1225,8 +1223,13 @@ def write(string):
             write("%30s = %s\n" % (setting, getattr(self.settings, setting)))
 
         temporary_file.flush()
+        self._call_pager(temporary_file)
+
+    def _call_pager(self, file_):
         pager = os.environ.get('PAGER', ranger.DEFAULT_PAGER)
-        self.run([pager, temporary_file.name])
+        # PAGER might contain arguments, like "less -R"
+        pager = pager.split(" ")
```
Comment (nfnty) [inline ranger/core/actions.py]: Use `shlex.split()`

Comment (Profpatsch) [inline ranger/core/actions.py]: Done.

File: ranger/core/actions.py [hunk 2]
```diff
@@ -1225,8 +1224,13 @@ def write(string):
...
 
         temporary_file.flush()
+        self._call_pager(temporary_file)
+
+    def _call_pager(self, file_):
         pager = os.environ.get('PAGER', ranger.DEFAULT_PAGER)
-        self.run([pager, temporary_file.name])
+        # PAGER might contain arguments, like "less -R"
+        pager = shlex.split(" ")
+        self.run(pager + [file_.name])
```
Comment (nfnty) [inline ranger/core/actions.py]: Should be `shlex.split(pager)`. Refactor into:
```
def run_pager(self, *paths):
    self.run(shlex.split(os.environ.get('PAGER', ranger.DEFAULT_PAGER)) + list(paths))
```

Comment (Profpatsch) [inline ranger/core/actions.py]: Right. Not sure what you mean by the refactor, though. I kind of like the split in three lines, this way the `split` can be documented.

Comment (Vifon) [inline ranger/core/actions.py]: +1, looks good to me in three lines. I'd be somewhat partial towards using a new variable name after the split but do what you think is right.

Comment (nfnty) [inline ranger/core/actions.py]: The refactor is for fixing the argument name, adding the ability to specify multiple paths and making it public for use.

This function is so simple that one line is more than enough, but either way works.

**Comment by nfnty** [comment]
Merged with changes in 79deadc7a9662807c075e3c61f3eb9362e29683c. Thanks.

**Pull request closed without merging**